### DISCUSSION TITLE
"Observation of the effect of gravity on the motion of antimatter"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#157] [Gravitation Models]
> Our results do not support cosmological models relying on repulsive matter–antimatter gravitation.

[https://www.nature.com/articles/s41586-023-06527-1](https://www.nature.com/articles/s41586-023-06527-1)

*"Antimatter embraces Earth, falling downward like normal matter: Study reveals gravity's effect on matter's elusive twin"*
[https://phys.org/news/2023-09-antimatter-embraces-earth-falling-downward.html](https://phys.org/news/2023-09-antimatter-embraces-earth-falling-downward.html)

**Comment by budrap00** [comment]
Another triumph of mathematicism. It has been demonstrated that a substance called antimatter that does not exist in physical reality outside of minute quantities "created" temporarily under highly constrained laboratory conditions does not have a property which would have accounted for antimatter's non-existence in empirical reality. What a shame. 

That means Dirac's equation predicting antimatter is still wrong as it always has been. The good news for the mathematicists is that they can now renew their quest to explain why something that exists in their model doesn't exist in physical reality - without having to concede that the dumb model is just wrong. No doubt generous funding for that feckless effort will be forthcoming.

**Reply by Francois-Zinserling** [reply]
The test merely eliminated a possibility that anti-matter might have anti-gravity effects. Not many expected it to have, but good to know for sure. 

Why should antimatter not exist? 
Positrons can be readily (although expensive) manufactured and already feature in medical devices. It annihilates beautifully predictable with an electron and is used in PET scans.

If you refer to the abominations they call anti-proton and anti-neutron then I am right there with you.

**Reply by RedshiftDrift** [reply]
Antiparticles, including anti-protons, are also produced naturally with every high energy collision of particles.  There is no mathematicism, those particles are just observed in physical reality and are just as stable as their "matter" counterpart.  The anti-hydrogen atoms respond to microwave signals just like hydrogen atoms do.

Antimatter is however unstable on contact with matter.  The collison produces photons.  That's why it's not observed in large quantity.

I posted the paper for its conclusion: "Our results do not support cosmological models relying on repulsive matter–antimatter gravitation."  The "repulsive gravitation" was the (ridiculous) hypothesis that is now rejected for sure.  The [matter-antimatter asymmetry is also a problem for the Big Bang model](https://home.cern/science/physics/matter-antimatter-asymmetry-problem).

It is of no use to cry *mathematicism* at every experiment you don't like, it just reveals your incapability to discriminate between good and bad experimental physics.

**Reply by bgreenman** [reply]
Right answer, repulsive matter-anti-matter gravitation is wrong, there must be another answer for the matter-anti-matter disparity

**Reply by HanDeBruijn** [reply]
>That means Dirac's equation predicting antimatter is still wrong as it always has been.

As a student, I've derived Dirac's equation myself (has cost me a few evenings) and I can assure you that there is not so much wrong with it. As confirmed by @RedshiftDrift , antimatter _does_ exist anyway. Only the [Dirac sea](https://en.wikipedia.org/wiki/Dirac_sea) is a questionable concept.

As for the experiment itself. Take a look at this picture from [Relativity Reexamined](http://www.alternatievewiskunde.nl/einstein/relativity-reexamined_compress.pdf) by [L&eacute;on Brillouin](https://en.wikipedia.org/wiki/L%C3%A9on_Brillouin).

![ACG](https://github.com/a-cosmology-group/acg/assets/135433222/12d74951-9127-4bd6-8ac2-8feb333fe1a9)

The experiment at hand has the second row II in this copy as an equivalent. It is seen that the acceleration $\gamma_2$ of the negative mass (antimatter) towards normal matter (the earth) is _positive_ as usual, despite of the fact that the force $f_2$ on it is _negative_. The earth itself does hardly move, of course. So proper theory shows no anti-gravitation for the situation II as well. **Wrong.** (Thanks @RedshiftDrift )

**Reply by budrap00** [reply]
To the extent that Dirac's math implies or predicts a 1 to 1 correspondence  between matter and so-called antimatter it is wrong. Antimatter in the sense of a full suite of antimatter atoms, elements and chemicals does not exist in empirical reality. The trivial amounts of positrons and anti-protons that appear briefly in highly constrained circumstances do not amount to a refutation of the statement that antimatter does not exist in the symmetric Diracian sense. Furthermore, even a rudimentary consideration of the consequences of a symmetric matter-antimatter distribution implies that the Cosmos would be constantly self-annihilating. Dirac's model is not only wrong it is unrealistic.

Here is where mathematicism enters the picture: "... _there must be another answer for the matter-anti-matter disparity._" There is no disparity in physical reality because antimatter is a mathematical consequence of a model that is clearly discordant with physical reality which does not contain any significant antimatter. The only "problem" lies with the theoretical expectation of a symmetrical matter-antimatter distribution. Asking for a physical explanation for that discordance gets the problem exactly backwards and produces _ad hoc_ half-assed conjectures like antigravity, that physicists then have to waste time disproving. 

Asking why there is a matter-antimatter asymmetry is like asking why the Sun goes around the Earth every day. The questions are physically meaningless because their assumptions about the nature of physical reality are fundamentally wrong.

In the similar case of dark matter, physicists have wasted decades chasing after something that is not there, all because theorists cannot wrap their mathematicist mind-set around the simple fact that Newtonian dynamics does not work well in systems much more complex than the solar system. The problem, as with antimatter, lies entirely with the mathematical models producing "expectations" that are discordant with reality. Mathematicism disposes contemporary theorists to consider the models they learned in school as axiomatically true, thereby creating and consigning themselves to the intellectual and scientific dead-end that is modern theoretical physics where their broken models rule over a desolate landscape.

**Reply by RedshiftDrift** [reply]
> The experiment at hand has the second row II in this copy as an equivalent... acceleration of the negative mass

No, the experiment realizes the first Row I.  Antimatter has a positive mass, from $E = mc^2$ and the energy that disappears from the matter particles in the accelerator to convert them into antimatter.  Only the charges of the particles are inverted.

The cases of Rows II, III and IV have never been experimentally observed.

**Reply by RedshiftDrift** [reply]
>  The only "problem" lies with the theoretical expectation of a symmetrical matter-antimatter distribution.

No, the symmetrical production of matter and antimatter is observed experimentally to high significance.  The problem with the *unexpected* matter-antimatter disparity is that it is based on the assumption that matter in the universe was created in a Big Bang.
If the idea of a Big Bang is abandoned, there is no matter-antimatter symmetry problem.

**Reply by budrap00** [reply]
`If the idea of a Big Bang is abandoned, there is no matter-antimatter symmetry problem.`

Well there is a Big Bang expectation but that is dependent on the Dirac model to some extent. So dropping the BB does not eliminate the Dirac model's expectation of a matter-antimatter symmetry.

**Reply by RedshiftDrift** [reply]
> Well there is a Big Bang expectation but that is dependent on the Dirac model to some extent.

No it's absolutely not, the BB expectation is only dependent on [empirical observations](https://github.com/orgs/a-cosmology-group/discussions/157#discussioncomment-7151703) that confirm the symmetry of matter-antimatter production.

> So dropping the BB does not eliminate the Dirac model's expectation of a matter-antimatter symmetry.

No, but dropping the BB eliminates the empirically expected matter-antimatter symmetry.

**Reply by budrap00** [reply]
`No it's absolutely not, the BB expectation is only dependent on [empirical observations](https://github.com/orgs/a-cosmology-group/discussions/157#discussioncomment-7151703) that confirm the symmetry of matter-antimatter production.`

Pair production in destructive collision events provides _inferential evidence only_ that pair production plays a role in whatever creation mechanism accounts for the widely distributed material elements that are observed throughout the Cosmos. There is absolutely no empirical evidence that provides support for the idea that a creation mechanism produced matter-antimatter pairs in abundance. That idea is refuted by the empirical observation that matter-antimatter symmetry is not broadly observed except under locally constrained circumstances. Conflating the destructive collision mechanism with a constructive creation mechanism is/was a categorical error. 

There is no empirical evidence that there is, could, or should be a matter-antimatter symmetry outside of the constrained circumstances under which it is (briefly) observed.

**Reply by RedshiftDrift** [reply]
I'm lost!  You use "Big Bang arguments" to show that "antimatter does not exist".  However, since "Big Bang arguments" are False, and $False \implies False$ and $False \implies True$ are both true statements, your **False Premise** fallacy has no bearing on the existence or not of antimatter.

**Reply by Francois-Zinserling** [reply]
I believe the matter-antimatter symmetry is a man-made catastrophe. It need not exist.

There was no charge symmetry problem at the time Dirac proposed antimatter in 1928. There still isn't. For every proton there is an electron and neutron charge =0. Pair production always have net charge =0. All collision events even at e.g. LHC can be reduced to a net charge of what went into the collisions.

Matter-antimatter symmetry problem arrived when we could not detect enough positrons in baryonic matter.

Solution: Get rid of fractionally charged quarks, find charge symmetry in the proton and neutron fundamental particles (not quarks), (electron is already fundamental) and the 'symmetry breaking' problem can be diminished to a particle energy difference - not a number count difference.

**Reply by budrap00** [reply]
I already stated in this discussion that antimatter does not exist in the sense of a full antimatter suite of particles, atoms, elements, chemistry, etc. The term antimatter implies much more than the occasional brief appearance of particle-antiparticle pairs that only occurs under highly constrained circumstances. There is no empirical evidence for the existence of antimatter in that broad sense and at best the limited observations of antiparticles is not empirical evidence of anything but the existence of antiparticles in the type of circumstances under which they appear. 

Empirical observations from which inferences are drawn _do not_ constitute empirical evidence of the things inferred. That misuse of the term empirical is what allows cosmologists to falsely claim that they have empirical evidence for the existence of dark matter, the BB, etc.

I should have been more clear that constructive creation events are a hypothesis. I do not know whether such events exist but our current state of knowledge does not preclude them. As you well know, I do not believe in the BB and therefore do not believe in its creation event. The creation event hypothesis is however independent of methodology. The argument is straightforward:

If there are, or were, creation event(s) that produce matter from energy and thus the Cosmos as we observe it, then those events cannot rely on pair production as seen in destructive collision events. Such creation events, by all empirical evidence must be net matter and not antimatter. If you want to propose an alternative hypothesis of no creation events just a fixed matter-energy balance, I cannot object; the question of which hypothesis is correct is an open one. Under neither scenario however, is it reasonable to posit the existence of a fundamental matter-antimatter symmetry for which there is absolutely no empirical evidence.

**Reply by RedshiftDrift** [reply]
Perhaps it's a matter of language, but when humans build someting and consistently observe it, I claim that **"it exists"**.

From where I sit right now, I can see from the window six houses other than the one I live in.  These houses are built by humans from matter and energy under highly constrained circumstances, and no such houses have been seen anywhere outside of Earth's surface.  Despite that, there is empirical evidence for the existence of houses, even if it's limited to the surface of the Earth and despite their brief appearances lasting around eighty years.

Now you could say that the existence of houses is only drawn from empirical observations of photons and *does not* constitute empirical evidence of houses.  I object to that wording because all our senses are through indirect inferences and such a description leads to solipsism without the possibility of any science.

We have to trust some inferences.  The presence and energy of particles revealed by a *detector pulse* is an inference that is just as valid as the houses I see with my eyes.  That's how we know that antimatter exists as much as houses do.

**Comment by bgreenman** [comment]
Anti-matter exists, its produced by accelerators like CERN. the reason Anti-matter dosen't produce anti-gravity is its the opposite version of matter,same matter opposite charges. if anti-gravity does exist its either a different force of naure or a third type of matter negative matter
